src/components/utils/DEPagination.js

```diff
@@ -171,8 +174,7 @@ function DEPagination(props) {
     const matches = useMediaQuery(theme.breakpoints.down("sm"));
     return (
         <Paper className={classes.paper}>
-            <Grid container>
-                <Grid item sm={4} md={5}></Grid>
+            <Grid container className={classes.mainGrid}>
```

Comment:
This works for now, but `Grid` container components have an equivalent `justify` prop that can be used instead:
```suggestion
            <Grid container justify="center">
```
This might be better, since the `Grid` component may be upgraded in the future to use CSS grid layouts, instead of the flex layout it's using under the hood now, and I'm not sure if the `justifyContent` CSS will still apply then.

FYI, the https://material-ui.com/components/grid/ page has an interactive demo that shows the effect of the different container prop settings.